dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkContext.java

```diff
@@ -6,8 +6,8 @@
 import javax.annotation.Nullable;
 
 public class UnitOfWorkContext {
-    private static final ThreadLocal<UnitOfWork> UNIT_OF_WORK = new ThreadLocal<>();
-    private static final ThreadLocal<SessionFactory> SESSION_FACTORY = new ThreadLocal<>();
+    public static final ThreadLocal<UnitOfWork> UNIT_OF_WORK = new ThreadLocal<>();
+    public static final ThreadLocal<SessionFactory> SESSION_FACTORY = new ThreadLocal<>();
 
```

Comment:
these need to exposed so we can setup the context for platform to use hibernate within a coroutine